docs/docs/unexpected-input.mdx

```diff
@@ -4,62 +4,63 @@ sidebar_label: Handling Unexpected Input
 title: Handling Unexpected Input
 ---
 
-import useBaseUrl from '@docusaurus/useBaseUrl';
+One thing you can count on when building a conversational assistant is that users
+will say unexpected things. Unexpected input is a deviation from the [`happy path`](glossary.mdx#happy_unhappy_paths)
+that you have defined.  For example:
 
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="handling-unexpected-user-input"></a>
+- A user walks away in the middle of a conversation about their subscription, then comes back
+and says "hi!"
+- A user asks "Why do you need to know that?" when the bot asks for their email address.
 
-All expected user inputs should be handled by the form we defined above, i.e. if the
-user provides the information the bot asks for. However, in real situations, the user
-will often behave differently. In this section we'll go through various forms of
-“interjections” and how to handle them within Rasa.
+This page is a guide on methods for handling unexpected input that is still within your bot's domain.
+Depending on what kind of unexpected
+input you're trying to handle, some or all of the methods describe may be applicable for you. This guide
+is not about disambiguating user input or handling out-of-scope questions; for these cases see
+the guide on [fallback and human handoff](fallback-handoff.mdx)
 
-The decision to handle these types of user input should always come from reviewing
-real conversations. You should first build part of your assistant, test it with real users
-(whether that's your end user, or your colleague) and then add what's missing. You shouldn't
-try to implement every possible edge case that you think might happen, because in the end
-your users may never actually behave in that way.
-[Rasa X](https://rasa.com/docs/rasa-x/)
-is a tool that can help you review conversations and make these types of decisions.
 
-## Generic interjections
+## User Interjections
 
-If you have generic interjections that should always have the same single response no
-matter the context, you can use the [Mapping Policy](./policies.mdx#mapping-policy) to handle these. It will always
-predict the same action for an intent, and when combined with a forgetting mechanism,
-you don't need to write any stories either.
+There are two kinds of unexpected input: generic interjections, and contextual interjections.
+Generic interjections are interruptions that should always get the same response regardless of the
+conversation context. [FAQs and chitchat](#chitchat-faqs) are common generic interjections.
+A contextual interjection is one whose response depends on the conversation context.
+For example, if a user asks "Why do you need that?", the answer will depend on what the bot just
+asked for.
 
-For example, let's say you see users having conversations like the following one with
+### Reverting Utterances
```

Comment:
with the rule policy, reverting utterances is no longer something we recommend so let's remove this section